### DISCUSSION TITLE
`bunx prisma migrate dev` outputs nothing

### DISCUSSION THREAD

**Discussion by alimulap** [prisma/prisma#25784] [Q&A]
### Question

Usually my project works perfectly fine, but now im trying to run it in a new machine (Ubuntu 20.04.6 LTS) and when i do `bunx prisma migrate dev` it's just outputs nothing.

Im sure the version is exactly the same as the repo, as i don't use the `^` for version. And the bun version is the same as my other machine.

### How to reproduce (optional)

1. Clone my project
3. Run `bun i`
4. Run `bunx prisma migrate dev`
5. Outputs nothing

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @alimulap 👋 

Can you try clearing your bun cache?
```
bun pm cache rm
```
Then retry the command again. if you still run into issues, you can try swapping out bun for npm and see if it works for you.

**Comment by alimulap** [comment]
thank you @ludralph for replying but turns out the problem was because i haven't installed nodejs on my machine. It wasn't clear at all because even with DEBUG=* i can't see anything useful. But after i want to try to use npm to build it i just realize i haven't installed nodejs on my machine.
